src/main/webapp/app/core/util/alert.service.ts

```diff
@@ -111,6 +111,12 @@ export class AlertService {
                     }
                     break;
 
+                case 403:
+                    // All access forbidden errors with additional identifier keys are handled by their own component
```

Comment:
Are you 100% sure that is the case **everywhere**? 🤔
Sadly, we have several different approach to handling server side errors, and some depend on this interceptor and expect it to spawn the alerts

Reply:
Not 100%, i will verify this tomorrow :+1: but since all errors with keys have an translation, there are probably handled in their component -> all errors without keys are handled by the interceptor -> so the theory :D

Reply:
Hmm, I *guess* we need to be very careful with that assumption :D
Especially components that are pretty old, e.g. stuff like the programming exercise details editor rely on this interceptor ...

Reply:
Yeah you are right - i guess it would be the best to revert that change and think about a current alternative which does not affect that much :+1:  (be6e4595e52be4fe2979eebbd631a3731f6d47c0)